tokio/src/net/mod.rs

```diff
@@ -33,10 +33,13 @@ cfg_net! {
 
     pub mod tcp;
     pub use tcp::listener::TcpListener;
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
I would consider making these annotations a macro. The reason being, it is easy to typo `"wasm32"` and not notice it.

Reply:
Another reason for using a macro is, if we end up *removing* the macro, we can delete the macro and then find the usages using the compiler. If we want to remove `#[cfg(not(target_arch = "wasm32"))` we then have to hunt through the code base.